src/AccessibilityInsights.Extensions.AzureDevOps/AzureDevOpsIntegration.cs

```diff
@@ -163,7 +166,7 @@ public void Disconnect()
         /// with the currently connected account
         /// </summary>
         /// <returns>Team projects associated with current account or null if disconnected</returns>
-        public IEnumerable<IProject> GetTeamProjects()
+        public IEnumerable<Models.TeamProject> GetTeamProjects()
```

Comment:
>public [](start = 8, length = 6)

Now that these methods are no longer called by the app, can we also tighten the visibility of these methods (or maybe the whole class) from public to internal?

Reply:
Class and functions changed to be internal